docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -297,8 +304,8 @@ If no attribute is selected, the app searches the list name by default.
 To export or delete an exception list, select the required action button on the appropriate list. Note that:
 
 * Exception lists are exported to `.ndjson` files.
-* Exception lists are also exported as part of any exported detection rules configured with exceptions. See <<import-export-rules-ui>>.
-* If a list is linked to any rules, you'll see a warning appear that asks you to confirm the deletion. If no rules are linked to a list, it is deleted without confirmation.
+* Exception lists are also exported as part of any exported detection rules configured with exceptions. Refer to <<import-export-rules-ui>>.
+* If a list is linked to any rules, you'll get a warning appear asking you to confirm the deletion. If no rules are linked to a list, it is deleted without confirmation.
```

Comment:
```suggestion
* If a list is linked to any rules, you'll get a warning asking you to confirm its deletion. If no rules are linked to a list, you can delete it without confirming.
```

Reply:
This might be overkill, but would it make sense here to add "exception" before "lists" to describe the list type? Also, I kind of prefer the original, though I'd make a couple minor tweaks:

```suggestion
* If a list is linked to any rules, you'll get a warning appear asking you to confirm the deletion. If no rules are linked, the list is automatically deleted without confirmation.
```

Reply:
@nastasha-solomon +1 on adding "exception" for clarity and consistency. For the second part, it feels a little redundant to me: I don't think we need both "automatically" and "without confirmation." Let's go with the latter, since you and @benironside are both leaning that way.
```suggestion
* If an exception list is linked to any rules, you'll get a warning appear asking you to confirm the deletion. If no rules are linked, the list is deleted without confirmation.
```